### PR TITLE
Tweaking release notes as per Ander

### PR DESCRIPTION
**Pull request by sfshaza2**
Incorporating feedback from Ander (https://github.com/flutter/devtools/pull/4845)

cc @polina-c

### CODE REVIEW

**Review by domesticmouse — APPROVED** [review]

File: src/development/tools/devtools/release-notes/release-notes-2.20.0-src.md [hunk 1]
```diff
@@ -1,43 +1,56 @@
...
 
-- Retire the Analysis tab - [#4714](https://github.com/flutter/devtools/pull/4714)
+- Retire the Analysis tab -
+  [#4714](https://github.com/flutter/devtools/pull/4714)
 
-- Add a new tab, Diff, to enable memory leak detection and troubleshooting
-by comparing heap snapshots, and providing insights about the number of
-instances, shallow size, retained size, and retaining
-paths - [#4714](https://github.com/flutter/devtools/pull/4714)
+- Add a new tab, Diff, to enable memory leak detection
```
Comment (atsansone) [inline src/development/tools/devtools/release-notes/release-notes-2.20.0-src.md:30]: **question:** Would we not put `Diff` in bold, per [GSG](https://developers.google.com/style/ui-elements#term-tab)?

File: src/development/tools/devtools/release-notes/release-notes-2.20.0-src.md [hunk 2]
```diff
@@ -1,43 +1,56 @@
...
+  and troubleshooting by comparing heap snapshots,
+  and providing insights about the number of instances,
+  shallow size, retained size, and retaining paths -
+  [#4714](https://github.com/flutter/devtools/pull/4714)
```
Comment (atsansone) [inline src/development/tools/devtools/release-notes/release-notes-2.20.0-src.md:34]: **question:** Does this need to be one sentence? It's also passive voice.

**suggestion:** Consider:
```suggestion
  and troubleshooting.
  The data returned on this page compares heap snapshots,
  and provides the number of instances, shallow size,
  retained size, and retaining paths. -
  [#4714](https://github.com/flutter/devtools/pull/4714)
```

**question:** Do we explain shallow and retained size? If not, are there better terms or phrases?

File: src/development/tools/devtools/release-notes/release-notes-2.20.0-src.md [hunk 3]
```diff
@@ -1,43 +1,56 @@
...
 
-- Add support for displaying coverage in CodeView - [#4700](https://github.com/flutter/devtools/pull/4700)
+- Add support for displaying coverage in CodeView -
+  [#4700](https://github.com/flutter/devtools/pull/4700)
 
   ![coverage]({{site.url}}/development/tools/devtools/release-notes/images-2.20.0/4700.png "coverage in CodeView")
 
 ## Network updates
 
 - Display request data if content type is not json
```
Comment (atsansone) [inline src/development/tools/devtools/release-notes/release-notes-2.20.0-src.md:54]: **issue:** This should be `if the content type is not in the JSON format.` per [GSG](https://developers.google.com/style/filenames#file-type-names).

**Review by atsansone — COMMENTED** [review]
I found a few issues and had a few small questions.

**Review by polina-c — APPROVED** [review]

**Review by khanhnwin — APPROVED** [review]

**Comment by sfshaza2** [comment]
Hi @atsansone! Generally, I do minimal-to-no editing on release notes. These are often autogenerated, and written by engineers. It's not worth our time to make them conform to all of the Google style guidelines. Especially because, _sometimes_ they'll resubmit them so all previous editing is "gone". It's not worth the trouble.

Thanks for the review.

**Pull request merged**